packages/katello/katello-repos/katello-repos.spec

```diff
@@ -2,12 +2,14 @@
 %global pulp_version 2.21
 %global use_pulp_nightly false
 
+%global pulpcore_version 3.4
```

Comment:
@jlsherrill or should we directly set 3.6 here? :)

Reply:
yeah, go straight to 3.6!